CMakeLists.txt

```diff
@@ -216,3 +242,15 @@ message(STATUS "Octave Templates (${ACADOS_OCTAVE_TEMPLATE})")
 message(STATUS "System name:version ${CMAKE_SYSTEM_NAME}:${CMAKE_SYSTEM_VERSION}")
 message(STATUS "Build type is ${CMAKE_BUILD_TYPE}")
 message(STATUS "Installation directory is ${CMAKE_INSTALL_PREFIX}")
+if(ACADOS_WITH_OPENMP)
+    message(STATUS "OpenMP parallelization is ON")
+    if(ACADOS_NUM_THREADS)
+        message(STATUS "OpenMP number of threads ${ACADOS_NUM_THREADS}")
+    else()
+        message(STATUS "OpenMP number of threads not set, will use omp_get_max_threads")
```

Comment:
That's the only remark: to be more specific staying something like "ACADOS_NUM_THREADS not set, will use omp_get_max_threads instead"

Reply:
Otherwise it can be merged.